Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM alpine:3.3
-MAINTAINER jp@roemer.im
+FROM alpine:3.5
+
+# Components versions
+ENV GOLANG_VERSION 1.8
+ENV GOLANG_SRC_URL https://golang.org/dl/go$GOLANG_VERSION.src.tar.gz
+ENV GOLANG_SRC_SHA256 406865f587b44be7092f206d73fc1de252600b79b3cacc587b74b5ef5c623596
+
+# Branch name
+#ENV GOGS_VERSION v0.9.141
+ENV GOGS_VERSION develop
```

Comment:
In the build-sh script I replaced the `go get ...` command by the `git clone ...` one (as underlined above) on purpose so that the desired Gogs version can be chosen.

Instead of relying on the local Gogs version, the one from where the Dockerfile is coming, this `git clone ...` allows to have it decoupled.

The Dockerfile is only using the content from the local `docker` folder, everything else, the whole gogs sources are coming from github while building the image.

So the GOGS_VERSION variable is then used to know the branch / tag to be cloned (only this branch / tag is retrieved to speedup the build).

My goal was to have a Docker build more or less independent from the local source tree, of course I can understand that this new behavior may not suit you.